python/ray/_private/runtime_env/utils.py

```diff
@@ -89,6 +89,10 @@ async def check_output_cmd(
         #     https://docs.python.org/3/library/asyncio-subprocess.html#asyncio.asyncio.subprocess.Process.stderr
         #   * Avoid mixing multiple outputs of concurrent cmds.
         stdout, _ = await proc.communicate()
+    except asyncio.exceptions.CancelledError as e:
```

Comment:
do we have any tests/repro steps to provide signals for this change?

Reply:
yess, I updated this because one of those test will fail without this in py 3.9. I forgot which one, let me remove this change and trace back which test needs this.

Reply:
@rickyyx this is the test that failed https://buildkite.com/ray-project/postmerge/builds/2431#018ceade-2b21-47b5-b554-1494daed53fe/6-1061; the runtime env is expected to timed out but is thrown as runtimeerror instead.